scripts/setup-k8s-master.sh.in

```diff
@@ -52,6 +52,10 @@ nodeName: ${HOSTNAME}
 tokenTTL: 0s
 apiServerCertSANs:
 - ${LB_DNS}
+controllerManagerExtraArgs:
+  address: 0.0.0.0
+schedulerExtraArgs:
+  address: 0.0.0.0
```

Comment:
Opening up the metrics endpoints without authentication for these makes me a bit uneasy. I think I would prefer if we held off on these updates until the auth changes land upstream.

Reply:
That's fine. I took this change out and left only the kubelet change for now.